ddtrace/internal/utils/http.py

```diff
@@ -422,20 +424,18 @@ class FormData:
     data: str
     content_type: str
 
-    def __str__(self):
-        return (
-            f'Content-Disposition: form-data; name="{self.name}"; filename="{self.filename}"\r\n'
-            + f"Content-Type: {self.content_type}\r\n"
-            + f"\r\n{self.data}\r\n"
-        )
 
+def multipart(parts: List[FormData]) -> Tuple[bytes, dict]:
+    msg = MIMEMultipart("form-data")
+    del msg["MIME-Version"]
```

Comment:
What's the reason for deleting "MIME-Version"? So it's not part of the body of the body or headers when we split it and then encode them? It's basically like, we like this object's functionality in all ways except it putting MIME-Version in there?

Reply:
Yep that's right. We don't need that header in our payloads, se we get rid of it.